docassemble_webapp/docassemble/webapp/server.py

```diff
@@ -2965,20 +2965,22 @@ def make_navbar(status, steps, show_login, chat_info, debug_mode, index_params,
     chat_sr = word("Live chat")
     source_message = word("Information for the developer")
     if debug_mode:
-        source_button = '<li class="nav-item d-none d-md-block"><button class="btn btn-link nav-link da-no-outline" title=' + json.dumps(source_message) + ' id="dasourcetoggle" data-bs-toggle="collapse" data-bs-target="#dasource"><i class="fas fa-code"></i></button></li>'
-        source_menu_item = '<a class="dropdown-item d-block d-md-none" title=' + json.dumps(source_message) + ' id="dasourcetoggle" href="#dasource" data-bs-toggle="collapse" aria-expanded="false" aria-controls="source">' + word('Source') + '</a>'
+        source_button = '<div class="nav-item navbar-nav d-none d-md-block"><button class="btn btn-link nav-link da-no-outline" title=' + json.dumps(source_message) + ' id="dasourcetoggle" data-bs-toggle="collapse" data-bs-target="#dasource"><i class="fas fa-code"></i></button></div>'
+        source_menu_item = '<a class="dropdown-item d-block d-md-none navbar" title=' + json.dumps(source_message) + ' id="dasourcetogglemenu" href="#dasource" data-bs-toggle="collapse" aria-expanded="false" aria-controls="source">' + word('Source') + '</a>'
```

Comment:
Is there any reason the `id="dasourcetogglemenu"` needs to be there? `dasourcetoggle` had an `id` because there is some JavaScript that toggles `active` on the parent element, but that isn't necessary for the menu item that small devices see, I don't think.

Reply:
I don't think it needs to be there. I changed it just to avoid two elements having the same ID which is invalid HTML, but removing it is a valid option too.